src/App.tsx

```diff
@@ -1,25 +1,19 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from 'react'
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch
+} from 'react-router-dom'
+import { Film, PageNotFound } from './routes'
```

Comment:
Interesting... Just looking at the files here on github, I'm not too sure how this works.
Importing Film and PNF from "./routes"?

Reply:
Noticed it's a folder now, oooops

Reply:
`./routes` contain an index.ts file which exports all exports of both `./routes/Film` and `./routes/PageNotFound`, and as `./routes` and `./routes/index` are "synonyms" we can use imports without knowing exactly which file they are from.